packages/round-manager/src/features/round/ViewRoundResults/ViewRoundResults.tsx

```diff
@@ -115,46 +130,66 @@ function useRevisedMatchingFunds(roundId: string, overridesFile?: File) {
 
 export default function ViewRoundResults() {
   const { chain } = useNetwork();
+  const { data: signer } = useSigner();
   const { id } = useParams();
   const navigate = useNavigate();
   const roundId = utils.getAddress(id as string);
+  const debugModeEnabled = useDebugMode();
+  const network = useNetwork();
+
+  const matchingTableRef = useRef<HTMLDivElement>(null);
   const [overridesFileDraft, setOverridesFileDraft] = useState<
     undefined | File
   >(undefined);
-  const [overridesFile, setOverridesFile] = useState<undefined | File>(
-    undefined
-  );
-  const matchingTableRef = useRef<HTMLDivElement>(null);
+  const [overridesFile, setOverridesFile] = useState<undefined | File>();
+
+  const [distributionOption, setDistributionOption] = useState<
+    "keep" | "scale"
+  >("keep");
 
   const {
     matches,
     isRevised: areMatchingFundsRevised,
     error: matchingFundsError,
     isLoading: isLoadingMatchingFunds,
     mutate: mutateMatchingFunds,
-  } = useRevisedMatchingFunds(roundId, overridesFile);
+  } = useRevisedMatchingFunds(
+    roundId,
+    distributionOption === "scale",
+    overridesFile
+  );
 
-  const debugModeEnabled = useDebugMode();
   const { data: round, isLoading: isLoadingRound } = useRound(roundId);
   const { round: oldRoundFromGraph } = useRoundById(
     (id as string).toLowerCase()
   );
-  const { data: signer } = useSigner();
+
   const isReadyForPayout = Boolean(
     oldRoundFromGraph?.payoutStrategy.isReadyForPayout
   );
-  const network = useNetwork();
   const matchToken =
     round &&
     payoutTokens.find(
       (t) => t.address.toLowerCase() == round.token.toLowerCase()
     );
 
-  const [distributionOption, setDistributionOption] = useState<
-    "keep" | "scale"
-  >("keep");
+  function formatUnits(value: bigint) {
+    return `${Number(utils.formatUnits(value, matchToken?.decimal)).toFixed(
+      4
+    )} ${matchToken?.name}`;
+  }
+
   const isBeforeRoundEndDate = round && new Date() < round.roundEndTime;
 
+  const sumOfMatches = useMemo(() => {
+    return (
+      matches?.reduce(
+        (acc: bigint, match) => acc + (match.revisedMatch ?? BigInt(0)),
```

Comment:
revised match should always be a bigint, if it's undefined we have a bigger problem here, we should investigate

Reply:
while this is valid - I am choosing violence and ignoring it because we need to role this out